Tests/Functional/xctest_checker/README.md

```diff
@@ -0,0 +1,13 @@
+# xctest_checker
+
+This tool is invoked by the swift-corelibs-xctest functional tests when they use the `{xctest_checker}` lit substitution. For more usage details, execute it from this directory using:
+
+```sh
+./xctest_checker.py -h
+```
+
+It includes unit tests which can be run with the following command in this directory:
+
+```sh
+PYTHONPATH=. python tests/test_compare.py
```

Comment:
Thanks for the tip!

I've updated the file, though I excluded the `-s` option because the context states that you'd be invoking the command from the correct directory already.